### DISCUSSION TITLE
Search results web part not seen for users from different region

### DISCUSSION THREAD

**Discussion by subbareddyvs** [microsoft-search/pnp-modern-search#1903] [Q&A]
Hi, 
The search page we have setup using the PnP webparts works fine for everyone, except users from different Region.  They do not get to see the Results web part at all in the page.

**Comment by subbareddyvs** [comment]
can you please help, below is the screenshot.  Only china based users get to see this issue.

![image](https://user-images.githubusercontent.com/63510269/156972082-61afad91-88cd-47fc-84f4-5b739c1cf05a.png)

**Comment by wobba** [comment]
It's a bit hard to assess what happens. I would start by opening dev tools and look for errors in the console log and/or network tab.

**Reply by subbareddyvs** [reply]
Hi,
it is the same as this one - https://github.com/microsoft-search/pnp-modern-search/issues/1749

**Reply by wobba** [reply]
If it's the same one, then it shouldn't be an issue with the latest release.
